test/Test/Db.hs

```diff
@@ -52,26 +57,34 @@ upsertReposSpec env = describe "upsertRepos" $ do
         env & reposRowDifference (upsertRepos [updatedValidRepo]) `equals` [Id 1 `WithId` updatedValidRepo]
 
 getReposSpec :: AppEnv -> Spec
-getReposSpec env = describe "getRepos" $
-    it "should return a list of repos with length 1" $ do
+getReposSpec env = describe "getRepos" $ do
+    it "should return a list of repos with length 1" $
         env & reposRowCount `equals` 1
-        env & reposUnaffected getRepos `equals` True 
+    it "should should not affect the repos table" $
+        env & reposUnaffected getRepos `equals` True
+
+getReposByCategoriesSpec :: AppEnv -> Spec
+getReposByCategoriesSpec env = describe "getReposByCategories" $ do
+    it "should return a list of repos with length 1" $
+        env & rowsReturned (getReposByCategories [Category "FFI"]) `equals` 1
```

Comment:
It seems that testing on comparison with the specific list of issues is more robust than just checking the length of the list
```suggestion
        env & getReposByCategories [Category "FFI"]) `equals` [testIssue]
```

Reply:
@chshersh Got it :+1: